### ISSUE TITLE
Tutorial Hints

### ISSUE THREAD

**Issue by ilegacy** [Acknex/LotterShock#70]
Was haltet ihr von Tutorial-Hints in Form von Text-Einblendungen a la "Press F" wenn man auf eine Tür zeigt bzw auf den PC? Ohne Erklärung hätte ich nie rausgefunden, dass man F drücken muss um die Türen zu öffnen. Ich dachte sie würden automatisch aufgehen und es wäre schade, wenn sich jemand das Spiel später runterlädt und daran scheitert.

**Comment by mk1x86** [comment]
Ja, sollten wir haben, so traurig das auch ist. Wäre glaube ich nicht so das Ding, wenn wir ein Optionsmenü hätten, wo man die Keys einstellen kann.

Man braucht auch eine Information, dass man mit den Jumpboots nen double jump machen kann, wobei das wohl auch Starlotti einfach sagen kann

**Comment by ilegacy** [comment]
Hm. Wenn man die Tasten nicht umschalten kann, können wir dann nicht einen hardcoded string ausgeben? Wir können auch direct auf die Türen bzw den Computer schreiben "Press USE" und dann in einer readme die controls erwähnen oder sowas...

**Comment by mk1x86** [comment]
ich würde das Fadenkreu ändern und da "USE" dranschreiben. Dass das Fadenkreuz grün wird, bekommt glaube ich keiner mit. Dafür ist es auch einfach zu klein

**Comment by Hen-woll** [comment]
Steffen und ich arbeiten an einem PanelTrigger der die steuerung stück für stück erklärt ! da dies einzelne bitmaps sind , ist eine key veränderung etwas blöd, und bringt mehr aufwand. 
gibt es eine alternative ?

**Comment by mk1x86** [comment]
am simpelsten wäre, ein Tastaturlayout anzuzeigen und an die Tasten dranzuschreiben, was sie machen.
Wenn man die umstellt, dann ist das halt so. Allerdings bin ich der Meinung, dass das Waffencyclen weg sollte und use auf "e" gelegt wird, wie es Standard ist.

**Comment by Hen-woll** [comment]
![keyboard](https://user-images.githubusercontent.com/30356887/42291903-fb85a230-7fce-11e8-8fac-83eb3083c7ed.png)

dies ist meine jetzige version, ich glaube hier meinen wir fast das selbe, ich hoffe das passt euch soweit.

**Comment by mk1x86** [comment]
joa, ich würde die Beschreibungen dann farbverschieben, ins orangene hinein.

**Comment by MasterQ32** [comment]
@Hen-woll ich würde in dem Bild die Controller-Belegung mit den [XBox-Symbolen](https://opengameart.org/content/xbox-360-controller-buttons) darstellen
Die Button-Belegung ist btw für nen XBox-Controller falsch, das muss
```
  Y
X   B
  A
```
sein

**Comment by Hen-woll** [comment]
Fixed.

**Comment by MasterQ32** [comment]
Die Lösung mit dem Overlay zu Spielbeginn ist grauenhaft. Zerstört jegliche Testability, Replayability und sonstige -bilities.

Fände es entweder als ersatz für den Ladebildschirm oder eben via Optionsmenü wesentlich besser...

**Comment by ilegacy** [comment]
@MasterQ32 Wenn der Screen im Ladebildschirm kommt (gute Idee!), wäre ich dafür, dass der Screen aber erst nach einem Tastendruck verschwindet. Ansonsten kann es sein, dass das Spiel sehr schnell lädt und man die Belegung nicht mehr lesen kann. Aber vielleicht könnte ein "Press Key to Continue..." auftauchen, sobald das Laden abgeschlossen ist?

Eine andere Sache: Es ist nicht ganz klar, was man mit den Jump-Boots bzw. dem Schutz-Anzug einsammelt. An diesen Stellen wären Tutorial-Hints gut, was das Item jetzt eigentlich genau macht. Wüsste ich nicht, was die Items machen, wäre ich nie drauf gekommen, dass man nun höher hüpfen kann bzw. in den Gift-Raum keinen Schaden mehr nimmt.

**Comment by mk1x86** [comment]
Siehe Storytexte im wiki. Das soll noch vertont werden. Wo ist der Schutzanzug?

**Comment by xwcg** [comment]
In einem der räume im neuen bereich den du mit der keycard aus der kantine freischaltest, da gibts nur 3 räume in dem bereich; wenn man sich zu doof anstellt die abzusuchen hat man pech gehabt

**Comment by mk1x86** [comment]
:-D ist ja gut. ich habe bisher nicht probiert, das Spiel regulär durchzuspielen

**Comment by ilegacy** [comment]
Der Schutzanzug ist derzeit ein sich drehener Kopf + Hände der in einem der Uniformen steckt die am Bett hängen. 

Es sind zwar nur 3 Räume aber man sollte nicht unterschätzen, dass man als Entwickler einen viel klareren Blick auf das Level hat. Ein neuer Spieler kann sich sehr einfach verirren und gerade bei uns sieht ja alles sehr ähnlich aus. Um ehrlich zu sein verlaufe ich mich ständig, muss immer wieder gucken, übersehe teils das Keypad, finde nicht sofort zu den richtigen Räumen und habe gar kein Gefühl welcher Bereich nun neu ist und welcher nicht. Das ist ein typisches Ding in der Entwicklung, dass die Entwickler denken "Ach ist doch alles easy" und kein normaler Spieler kommt am Ende damit klar. Bin gespannt auf Marius Feedback wenn er es das erste mal "richtig" spielt.

**Comment by mk1x86** [comment]
Oh, ich finde mich eigentlich recht gut zurecht. Die Gänge müssen halt mehr Krams an den Wänden haben, um das zu unterscheiden.
Das soll also der Schutzanzug sein. Kein Wunder, dass ich ihn nicht erkannt habe ^^
@ilegacy wie schwer ist denn so ein Schutzanzug zu basteln?

**Comment by MasterQ32** [comment]
Schade, dass man die `marine.mdl` aus `shadertest` nicht verwenden darf.

Bzgl Orientierung: Es gibt eine Minimap, die man mit Select bzw. Tab einblenden kann

**Comment by mk1x86** [comment]
gute Idee

**Comment by ilegacy** [comment]
Wer kann das machen? Ich könnte es, wenn es ein FBX dazu gibt. Ansonsten vielleicht @MasterQ32 ? Evtl geht sowas ja direkt mit den Engine-Tools?

**Comment by MasterQ32** [comment]
Das Spieler-Modell ist ja komplett in MED erstellt... Ich bin da aber sehr unbegabt, was modellieren angeht, das müsste @Hen-woll übernehmen

**Comment by xwcg** [comment]
ohne kopf ist so nen schutzanzug aber recht ungeschützt xD

**Comment by mk1x86** [comment]
Frag mal Gordon Freeman, der musste damit auch in Kältekammern usw. ;)

**Comment by Hen-woll** [comment]
Done.
@xwcg GitHub\LotterShock\Game\Assets [Rüstung.Mdl]

![screenshot 2018-07-06 21 51 45](https://user-images.githubusercontent.com/30356887/42397668-13afb280-8167-11e8-813e-dd6cb537460d.png)

**Comment by xwcg** [comment]
danke werde das morgen einsetzen!
